docs/3-scaling/0-conduit-operator.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: "Conduit Operator"
+toc_max_heading_level: 6
+---
+
+The **Conduit Operator** is a [Kubernetes operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) designed to simplify the management and orchestration of Conduit instances. The operator extends the [Kubernetes API](https://kubernetes.io/docs/concepts/overview/kubernetes-api/) with a [custom resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) for Conduit. 
+
+This custom resource allows users to define and manage Conduit instances declaratively, enabling seamless integration with Kubernetes' native features such as scaling, monitoring, and logging.
+
+### Overview
+
+Conduit pipelines are represented as Conduit custom resources, where each pipeline will be provisioned 
+as a distinct Conduit instance with its own lifecycle. 
+
+The Conduit custom resource definition format is very similar to that of [pipeline configurations](/docs/using/pipelines/configuration-file):
+
+```yaml
+apiVersion: operator.conduit.io/v1
```

Comment:
This doesn't seem quite right, the version should be `v1alpha`. I believe up-to-date examples are in `config/samples`.

cc @lyuboxa